### DISCUSSION TITLE
doc: vitest config in vite.config.js

### DISCUSSION THREAD

**Discussion by alexzhang1030** [vitest-dev/vitest#1017] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/49969959/159929589-2ead8381-0156-4e3c-87cd-0abebec5e6b6.png)

please look at the picture, i do not understand the example is in `vitest.config.js` or `vite.config.js`, does anybody can answer me? thanks a lot

**Comment by sheremet-va** [comment, marked as answer]
If your project uses Vite, you can extend your Vite config.
If not, you can either create `vite.config` or `vitest.config`

**Reply by alexzhang1030** [reply]
thanks a lot

**Reply by alexzhang1030** [reply]
so does i really need `triple slash command` to configure vitest itself in `vite.config` ?

**Reply by sheremet-va** [reply]
> so does i really need `triple slash command` to configure vitest itself in `vite.config` ?

`/// <reference ` is needed only if you are using `defineConfig` from Vite itself, since we are merging the types ourselves. You don't need it, if you use `defineConfig` from `vitest/config`

Basically it's needed only for typing of `test` property on your config

**Reply by alexzhang1030** [reply]
got it ! thank u !
